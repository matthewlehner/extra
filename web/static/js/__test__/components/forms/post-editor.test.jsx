// @flow
import React from "react";
import renderer from "react-test-renderer";

import PostEditor from "../../../components/forms/post-editor";

describe("<PostEditor />", () => {
  const props = { channels: [], post: { body: {}, channels: [] } };

  it("renders without crashing", () => {
    const tree = renderer.create(<PostEditor {...props} />);

    expect(tree).toMatchSnapshot();
  });
});
