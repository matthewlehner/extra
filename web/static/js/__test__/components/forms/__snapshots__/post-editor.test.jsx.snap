// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostEditor /> renders without crashing 1`] = `
<form
  onSubmit={[Function]}
>
  <dl
    className="form__control-group"
  >
    <dt
      className="label"
    >
      Collection
    </dt>
    <dd
      className="inputContainer"
    >
      Collection Name
    </dd>
  </dl>
  <div
    className="form__control-group"
  >
    <label
      className="label"
      htmlFor="post-content"
    >
      Content
    </label>
    <textarea
      className="form__control form_textarea"
      id="body"
      name="body"
      onChange={[Function]}
      required={true}
      rows="3"
      value={Object {}}
    />
  </div>
  <div
    className="form__control-group"
  >
    <span
      className="label"
    >
      Channels
    </span>
    <div
      className="form__collection-wrapper"
    />
  </div>
  <div
    className="form__actions"
  >
    <button
      className="button button_cancel"
      disabled={false}
      onClick={undefined}
    >
      Cancel
    </button>
    <button
      className="button"
      disabled={false}
      type="submit"
    >
      Create Post
    </button>
  </div>
</form>
`;
